app/models/game.rb

```diff
@@ -0,0 +1,5 @@
+class Game < ApplicationRecord
+  has_one :white_player
```

Comment:
For each of these, Rails won't know which model to use automatically. So you'll need to do this

`has_one :white_player, class_name: 'User'`